src/Google.Api.Gax.Grpc.Gcp/Google.Api.Gax.Grpc.Gcp.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\CommonProjectProperties.xml" />
+
+  <!-- For the moment, this package is versioned independently. -->
+  <PropertyGroup>
+    <Version>2.5.0-beta01</Version>
```

Comment:
Just checking that you want to start this at v2.5.0? If it's independently versioned, why not start at 1.0.0?

Reply:
It was deliberate - while it's beta it'll be independent, but I *think* we want to get it in sync with all the rest.